deploy/role_binding.yaml

```diff
@@ -2,48 +2,14 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: ibm-iam-operator
+  labels:
+    app.kubernetes.io/instance: ibm-iam-operator
+    app.kubernetes.io/managed-by: ibm-iam-operator
+    app.kubernetes.io/name: ibm-iam-operator
 subjects:
 - kind: ServiceAccount
   name: ibm-iam-operator
 roleRef:
   kind: Role
   name: ibm-iam-operator
   apiGroup: rbac.authorization.k8s.io
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: ibm-iam-operator
-subjects:
-- kind: ServiceAccount
-  name: ibm-iam-operator
-  namespace: ibm-common-services
-roleRef:
-    kind: ClusterRole
-    name: ibm-iam-operator
-    apiGroup: rbac.authorization.k8s.io
----
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: ibm-iam-operand
-subjects:
-- kind: ServiceAccount
-  name: ibm-iam-operand
-roleRef:
-  kind: Role
-  name: ibm-iam-operand
-  apiGroup: rbac.authorization.k8s.io
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: ibm-iam-operand
-subjects:
-- kind: ServiceAccount
-  name: ibm-iam-operand
-  namespace: ibm-common-services
-roleRef:
-    kind: ClusterRole
-    name: ibm-iam-operand
-    apiGroup: rbac.authorization.k8s.io
```

Comment:
Is it mandatory to separate these into multiple files? Auto generation of csv may be come a problem.